backend/src/app.ts

```diff
@@ -12,14 +13,24 @@ app.use(express.json());
 
 app.use(router);
 
-// Express does not pick up this method if next is not last parameter even if it is unused
-// eslint-disable-next-line @typescript-eslint/no-unused-vars
-app.use((err: Error | { error: Error; status: number }, req: Request, res: Response, next: NextFunction) => {
-  if (err instanceof Error) {
-    return res.status(status.INTERNAL_SERVER_ERROR).json({ error: err.message });
-  }
-  res.status(err.status).json({ error: err.error.message });
-});
+app.use(
+  (
+    err: Error | { error: Error; status: number } | ValidationError,
+    req: Request,
+    res: Response,
+    // Express does not pick up this method if next is not last parameter even if it is unused
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    next: NextFunction,
```

Comment:
Could this have been written like this:
```js
_next: NextFunction,
```
so that `eslint` knows it is not used? The underscore should mean that it is not used.

Reply:
<img width="633" alt="image" src="https://user-images.githubusercontent.com/32748145/180720365-2eba895f-0cec-4ca1-aa05-9cac50f0c2df.png">

still the same warning